packages/cli/src/config/config.ts

```diff
@@ -186,7 +186,11 @@ export async function loadCliConfig(
   sessionId: string,
 ): Promise<Config> {
   const argv = await parseArguments();
-  const debugMode = argv.debug || false;
+  const debugMode =
+    argv.debug ||
+    [process.env.DEBUG, process.env.DEBUG_MODE].some(
```

Comment:
I know you didn't add these but why are there two, is this an us thing or a node thing?